task11.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'str-utils' {
```

Comment:
The solution is correct although the rest of the code is missing in order to run it.